src/libraries/System.Globalization/tests/CultureInfo/GetCultureInfo.cs

```diff
@@ -0,0 +1,47 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using Xunit;
+
+namespace System.Globalization.Tests
+{
+    public class GetCultureInfoTests
+    {
+        public static bool PlatformSupportFakeCulture => !PlatformDetection.IsWindows || (PlatformDetection.WindowsVersion >= 10 && !PlatformDetection.IsFullFramework);
+
+        [ConditionalTheory(nameof(PlatformSupportFakeCulture))]
+        [InlineData("en")]
+        [InlineData("en-US")]
+        [InlineData("ja-JP")]
+        [InlineData("ar-SA")]
+        [InlineData("xx-XX")]
+        public void GetCultureInfo(string name)
+        {
+            Assert.Equal(name, CultureInfo.GetCultureInfo(name).Name);
+            Assert.Equal(name, CultureInfo.GetCultureInfo(name, predefinedOnly: false).Name);
+        }
+
+        [ConditionalTheory(nameof(PlatformSupportFakeCulture))]
+        [InlineData("en@US")]
+        [InlineData("\uFFFF")]
+        public void TestInvalidCultureNames(string name)
```

Comment:
Empty string?

Reply:
Empty string is a valid culture name for the Invariant culture.